locations/spiders/krispy_kreme_mx.py

```diff
@@ -0,0 +1,23 @@
+from scrapy import Spider
+
+from locations.categories import Categories
+from locations.items import Feature
+from locations.pipelines.address_clean_up import clean_address
+
+
+class KrispyKremeMXSpider(Spider):
+    name = "krispy_kreme_mx"
+    item_attributes = {"brand": "Krispy Kreme", "brand_wikidata": "Q1192805", "extras": Categories.FAST_FOOD.value}
+    start_urls = ["https://www.krispykreme.mx/directorio-tiendas/"]
+
+    def parse(self, response):
+        locations = response.xpath('//tbody[@id="myTable"]/tr')
+        for index in range(0, len(locations)):
+            item = {}
+            item["ref"] = f"krispy_kreme_mx_{index}"
```

Comment:
Instead of generating fake refs, can you set `no_refs = True` under `start_urls` above?